forms-flow-components/src/components/CustomComponents/Button.tsx

```diff
@@ -0,0 +1,136 @@
+import { useRef, useEffect, useState, ReactElement } from "react";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import Dropdown from "react-bootstrap/Dropdown";
+import { ChevronIcon } from "../SvgIcons/index";
+
+interface DropdownItem {
+  label: string;
+  onClick: () => void;
+  dataTestid?: string;
+  ariaLabel?: string;
+}
+
+interface CustomButtonProps {
+  variant: string;
+  size?: "sm" | "lg" ;
+  label: string;
+  onClick?: () => void;
+  isDropdown?: boolean;
+  dropdownItems?: DropdownItem[];
+  disabled?: boolean;
+  icon?: React.ReactNode;
```

Comment:
can we give icon position mabe some icon will come start some will come on end @arun-s-aot

Reply:
For now, icon inside button is always positioned in left side with current design system . 
If there is a design with a different approach, we can plan to add it later Shuhaib .

Reply:
okay